src/Controls/src/Core/HandlerImpl/Button/Button.Windows.cs

```diff
@@ -5,11 +5,37 @@
 using Microsoft.Maui.Controls.Internals;
 using Microsoft.Maui.Controls.Platform;
 using Microsoft.Maui.Handlers;
+using Microsoft.UI.Xaml;
+using WButton = Microsoft.UI.Xaml.Controls.Button;
 
 namespace Microsoft.Maui.Controls
 {
 	public partial class Button
 	{
+		WButton _button;
+
+		private protected override void OnHandlerChangedCore()
```

Comment:
Drop private

Reply:
It is inherited, in order to do it I would need to change the virtual method in` Element.Impl.cs` and the rest of the overrides.